src/mapml/layers/MapLayer.js

```diff
@@ -425,7 +425,21 @@ export var MapMLLayer = L.Layer.extend({
         //L.DomEvent.disableClickPropagation(removeControlButton);
         L.DomEvent.on(removeControlButton, 'click', L.DomEvent.stop);
         L.DomEvent.on(removeControlButton, 'click', (e)=>{
+          let fieldset = 0, elem, root;
+          root = mapEl.tagName === "MAPML-VIEWER" ? mapEl.shadowRoot : mapEl.querySelector(".mapml-web-map").shadowRoot;
+          if(e.target.closest("fieldset").nextElementSibling && !e.target.closest("fieldset").nextElementSibling.disbaled){
+            elem = e.target.closest("fieldset").previousElementSibling;
+            while(elem){
+              fieldset += 2; // find the next layer menu item
+              elem = elem.previousElementSibling;
+            }
+          } else {
+            // focus on the link
+            elem = "link";
+          }
           mapEl.removeChild(e.target.closest("fieldset").querySelector("span").layer._layerEl);
+          elem = elem ? root.querySelector(".leaflet-control-attribution").firstElementChild: elem = root.querySelectorAll('input')[fieldset];
+          setTimeout(() => elem.focus(), 800); // a timeout is set so "pressed remove layer" is announced first
```

Comment:
This timeout seems to cause NVDA (using Chrome) to announce the document title + map then next layer/link, I'm not hearing "pressed remove layer". In that sense I think doing without the timeout provides a better experience?

Reply:
Using a Mac and voiceover, I hear "pressed remove layer", but if that's the case with NVDA, I could remove the timeout for better consistency

Reply:
> if that's the case with NVDA, I could remove the timeout for better consistency

There doesn't seem to be a whole lot of guidance on the desired behavior when removing/hiding the currently focused element. But I think removing the timeout provides a similar experience to when e.g. dialog elements are hidden as a result of closing it: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog#result